src/docs/monarch-roles/ontology_lead.md

```diff
@@ -0,0 +1,13 @@
+## Monarch role: Ontology lead
```

Comment:
Dear @matentzn, 

This should now be called the "Monarch Ontology Technical Lead" instead. Please update. 

The "Curator Lead" will now be called the "Ontology Curation Lead."

This will help remove some of the ambiguity around the activities to report during huddles. 
I already shared this information with both @sabrinatoro and @twhetzel.

Reply:
Renamed!